components/checkbox/index.en-US.md

```diff
@@ -0,0 +1,31 @@
+---
+category: Components
+type: Form Controls
+title: Checkbox
+---
+
+Checkbox.
+
+## When To Use
+
+- Used to select serveral values in multiple options.
+- If you just use only one checkbox, it is the same as `Switch` to switch between two states. The difference is that `Switch` will trigger state change directly, but `Checkbox` just marks state changed and need to be submitted.
+
+## API
+
+### Checkbox
+
+| Property      | Description             | Type      | Default  |
+|----------|------------------|----------|--------|
+| checked | Specifies whether the current is selected | Boolean  | false |
+| defaultChecked | Initial whether or not selected | Boolean | false |
```

Comment:
| defaultChecked | Specifies the initial state: whether or not the checkbox is selected. | Boolean | false |